src/h5web/toolbar/controls/DomainSlider/BoundEditor.tsx

```diff
@@ -0,0 +1,85 @@
+import { ReactElement, useEffect, useRef, useState } from 'react';
+import { FiCheck, FiSlash } from 'react-icons/fi';
+import { formatValue } from '../../../utils';
+import styles from './BoundEditor.module.css';
+
+interface Props {
+  label: string;
+  value: number;
+  isEditing: boolean;
+  hasError: boolean;
+  onEditToggle: (force: boolean) => void;
+  onChange: (val: number) => void;
+}
+
+function BoundEditor(props: Props): ReactElement {
+  const { label, value, isEditing, hasError, onEditToggle, onChange } = props;
+
+  const id = `${label}-bound`;
+  const inputRef = useRef<HTMLInputElement>(null);
+
+  const [inputValue, setInputValue] = useState(formatValue(value));
+
+  useEffect(() => {
+    setInputValue(isEditing ? value.toString() : formatValue(value));
+  }, [isEditing, value, setInputValue]);
+
+  useEffect(() => {
+    if (!isEditing) {
+      // Remove focus from min field when editing is turned off
+      inputRef.current?.blur();
+    }
+
+    if (isEditing && label === 'Min') {
+      // Give focus to min field when opening tooltip in edit mode
+      inputRef.current?.focus();
+    }
+  }, [isEditing, label]);
+
+  return (
+    <form
+      className={styles.boundEditor}
+      data-error={hasError || undefined}
+      data-editing={isEditing}
+      onSubmit={(evt) => {
+        evt.preventDefault();
+        onChange(Number.parseFloat(inputValue));
+        onEditToggle(false);
+      }}
+    >
+      <label id={`${id}-label`} className={styles.label} htmlFor={id}>
+        <abbr title={value.toString()}>{label}</abbr>
```

Comment:
I find counter-intuitive to have the `value` on the label.

Isn't there a way to have it on hover on the editing box when not editing ?

Reply:
I'll give it a try.

Reply:
Alright, as I thought, I can just use the `title` attribute on the input field. The downside is that there no longer is an indication that supplementary information is available like there was with `abbr` (the dotted underline). Not a huge problem.

![image](https://user-images.githubusercontent.com/2936402/110782400-5ee15480-8267-11eb-8366-156a393b36a2.png)